packages/frontpage/lib/auth.ts

```diff
@@ -303,9 +308,23 @@ export const handlers = {
         throw new Error("Missing expires");
       }
 
+      const subjectDid = tokensResult.data.sub
+        ? parseDid(tokensResult.data.sub)
```

Comment:
Should add a comment here explaining that this is safe because in the case where we are expecting a specific DID, we already check if the sub matches above.